src/packages/solid/components/ProgressBar/ProgressBar.tsx

```diff
@@ -38,17 +38,22 @@ export interface ProgressBarProps extends UIComponentProps {
    * total height of the component
    */
   height?: number;
+
+  toned?: boolean;
+
+  toneStyles?: Omit<ProgressBarStyles, 'tone'>;
 }
 
 const ProgressBar: Component<ProgressBarProps> = (props: ProgressBarProps) => {
+  const toneStyles = props.toneStyles;
   return (
     <View
       {...props}
       forwardStates
       // @ts-expect-error TODO type needs to be fixed in framework
       style={[
         props.style, //
-        styles.Container.tones[props.tone || styles.tone],
+        toneStyles?.Container,
```

Comment:
You don't need to pass in toneStyles - just remove this all together. When you wrap the component you're just passing in props.style